python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -161,6 +163,174 @@ def run(self, dataslice, slicePoint=None):
             result = self.badval
         return result
 
+
+class ParallaxCoverageMetric(BaseMetric):
+    """
+    Check how well the parallax factor is distributed. Subtracts the weighted mean position of the
+    parallax offsets, then computes the weighted mean radius of the points.
+    If points are well distributed, the mean radius will be near 1. If phase coverage is bad,
+    radius will be close to zero.
+
+    For points on the Ecliptic, uniform sampling should result in a metric value of ~0.5.
+    At the poles, uniform sampling would result in a metric value of ~1.
```

Comment:
It sounds like the results of this metric are pretty complicated, if they depend on ecliptic latitude. Could the resulting value be weighted by ecliptic latitude (we have the ra/dec of the slicepoint or field .. this could be converted to ecliptic latitude pretty easily) to make have a uniform range across the sky?